src/test/lib/scheduler/scheduler_test.cpp

```diff
@@ -126,113 +131,126 @@ TEST_F(SchedulerTest, BasicTest) {
   Hyrise::get().topology.use_fake_numa_topology(8, 4);
   Hyrise::get().set_scheduler(std::make_shared<NodeQueueScheduler>());
 
-  std::atomic_uint32_t counter{0};
+  auto counter = std::atomic_uint32_t{0};
 
   increment_counter_in_subtasks(counter);
 
   Hyrise::get().scheduler()->finish();
 
-  ASSERT_EQ(counter, 30u);
-
-  Hyrise::get().set_scheduler(std::make_shared<ImmediateExecutionScheduler>());
+  ASSERT_EQ(counter, 30);
```

Comment:
```suggestion
  EXPECT_EQ(counter, 30);
```

Reply:
(also for the tests below)

Reply:
Just a note: when I add your comment to the batch and commit the suggestions, this subthread here is resolved. Thus, your second comment is hidden.